features/upgrade/routing/upgrade.feature

```diff
@@ -89,3 +89,91 @@ Feature: Routing and DNS related scenarios
     Then the expression should be true> deployment("router-default").current_replicas(cached: false) == <%= cb.num_workers %>
     And <%= cb.num_workers %> pods become ready with labels:
       | ingresscontroller.operator.openshift.io/deployment-ingresscontroller=default |
+
+
+  # @author hongli@redhat.com
+  # @case_id OCP-38812
```

Comment:
I think we do not use case ID for prepare.

Reply:
Thank you. Yes, just keep the case ID for upgrade-check.